lib/student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Since you are calling the `super` method here and the `parent_permission` method from the parent class is a **_keyword_** argument, you ought to pass it a value of the `parent_permission` parameter that is being created for the `Student` class. This will map the variables properly and the `super` method will use the conditions set in the `initialize` method of the `Person` class to be the same condition as those passed as parameters in the `super` method. Kindly implement this change so that you don't get these errors 💻 🖱️:

![image](https://github.com/Grandi0z/school_library/assets/77106962/ec0fe63b-6f6d-495b-ad5b-d466720438fb)

#### Hint;

```ruby
  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission)
    @classroom = classroom
  end
```